outputs/core_pools.json

```diff
@@ -88,5 +95,9 @@
     "0x0c659734f1eef9c63b7ebdf78a164cdd745586db000000000000000000000046": "USDC/USDbC/axlUSDC"
   },
   "mode": {},
-  "fraxtal": {}
+  "fraxtal": {
```

Comment:
Fraxtal is a new chain/no fee sweeping yet.  Will review core pools when we do first fee round.